 Martin V\"ath <martin at mvath.de>:
 The parsing in meson.build does not match to the provided client.h file.
 This is an mpv-0.36.? (upstream) bug.
 For simplicity, we just patch in the version which is current with 0.36.?.
--- 1/meson.build
+++ 1/meson.build
@@ -1709,8 +1709,8 @@
 
 client_h_define = cc.get_define('MPV_CLIENT_API_VERSION', prefix: '#include "libmpv/client.h"',
                                 include_directories: include_directories('.'))
-major = client_h_define.split('|')[0].split('<<')[0].strip('() ')
-minor = client_h_define.split('|')[1].strip('() ')
+major = '2'
+minor = '1'
 client_api_version = major + '.' + minor + '.0'
 
 libmpv = library('mpv', sources, dependencies: dependencies, gnu_symbol_visibility: 'hidden',
